src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -225,6 +233,18 @@ private void setDefaults() {
 			usage("please specify --output_file=FILE to use the file producer");
 		}
 
+		if ( this.bootstrapperType == null ) {
+			this.bootstrapperType = "async";
+		} else if ( !this.bootstrapperType.equals("async")
+				&& !this.bootstrapperType.equals("sync")
+				&& !this.bootstrapperType.equals("none") ) {
+			usage("please specify --bootstrapper=async|sync|none");
+		}
+
+		if ( this.bootstrapperBatchFetchSize  == null ) {
+			this.bootstrapperBatchFetchSize = 64000;
```

Comment:
it looks like we're never consuming the value of the `bootstrapper_fetch_size` flag.  But more than that, it's kinda unclear to me that the subsequent `Statement#setFetchSize` call actually does the right thing.  Have you tested this with really big tables to ensure that we're not buffering the entire table into memory?